nuget/Microsoft.Windows.CppWinRT.targets

```diff
@@ -828,7 +828,7 @@ $(XamlMetaDataProviderPch)
     <!--Append any additional item metadata after all default and project settings have been applied-->
     <ItemDefinitionGroup>
       <ClCompile>
-        <AdditionalOptions>%(AdditionalOptions) /bigobj /await</AdditionalOptions>
```

Comment:
I was about to submit a PR that adds this below here:

```
        <AdditionalOptions Condition="'%(ClCompile.LanguageStandard)' == 'stdcpp17'">%(AdditionalOptions) /await</AdditionalOptions>
```

That would keep the behavior the same for folks who still target stdcpp17 and anyone who kept the defaults from the project templates.

Reply:
Feel free to add it to this branch or just submit your branch for PR and we can scrap this PR.

Reply:
Let me know if I got that right.